default/stringtables/en.txt

```diff
@@ -1974,6 +1974,9 @@ Enable autosave when resigning or closing the game. In multiplayer hostless mode
 OPTIONS_DB_AUTOSAVE_HOSTLESS
 Enable autosave in hostless mode.
 
+OPTIONS_DB_AUTOSAVE_HOSTLESS_EACH_PLAYER
+Enable autosave in hostless mode after each player issued orders. Prevents losing player orders on server crash.
```

Comment:
Adding more text about motivation doesn't clarify what "after each player issued orders" means.

Reply:
It means after player issued orders, server saves game to don't lost orders in case of crash.

Reply:
"after player issued orders" is unclear.

Reply:
When player presses Turn button and sends `TURN_ORDERS` message.

Reply:
I think it means to make an auto-save every time a player sends orders to server by pushing the "Turn" button". @o01eg, please confirm if this is what you mean.

Reply:
@Oberlus Yep, that what I mean.